GolfExample_TCAPI/index.html

```diff
@@ -41,12 +39,75 @@
                 }
             );
 
-        tincan.sendStatement(
-            {
-                verb: "attempted",
-                context: GolfExample.getContext()
+        var bookmarkingTracker = function(){
```

Comment:
It is common in JavaScript to capitalize the constructor function to show that it should be used with `new` as opposed to standalone like other functions. This is a nit pick but if you want to fix it I'd be happy :-).